apps/cyberstorm-remix/app/commonComponents/Footer/FooterLinks.css

```
@@ -0,0 +1,33 @@
+
+    @container footer (width >= 60rem) {
+      flex: 1;
+    }
+  }
+
+  .footer-links__section {
+    display: grid;
+    grid-template-rows: auto 1fr;
+    gap: 1.5rem;
+
+    min-width: 20ch;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)